Home_work14/Home_work14.cpp

```diff
@@ -1,32 +1,42 @@
 ﻿#include <iostream>
 #include <string>
+#include <vector>
+#include <algorithm>
+
+class Veapon; class Player; class Team;
 enum class Role {
 	Assault,
 	Support,
 	Sniper,
 	Medic
 };
 class Rectangle {
+private:
+	float lenght{}, hight{};
```

Comment:
please validate proper names spelling to not confuse other people who may potentially use your class.